pandas/_libs/src/ujson/lib/ultrajson.h

```diff
@@ -108,11 +108,7 @@ typedef uint32_t JSUINT32;
 
 #define FASTCALL_MSVC
 
-#if !defined __x86_64__ && !defined __aarch64__
-#define FASTCALL_ATTR __attribute__((fastcall))
-#else
 #define FASTCALL_ATTR
```

Comment:
If you can remove this altogether would be preferable. No need for an empty define

Reply:
Removing the empty defines also needs lot of cleanup in ultrajsondec.c . Otherwise there will be compilation problems. And also not sure about the fastcall defines under _WIN32 .